### ISSUE TITLE
Interactive statistics

### ISSUE THREAD

**Issue by Deleetdk** [openpsych/django#16]
1. Set up a Shiny (R) server on the server.
2. Make a Django view that embeds this Shiny.

On the Shiny have things like:

- Number of views per publication. Calculate this data from the apache log.
- Review time distribution. Color by journal. Summary statistics.
- Publications by person.
- Reviews by person.

Regarding the data from the apache log. One could calculate this on the fly, but this doesn't seem like a good idea as the log grows in size. Better to extract the information once a day/hour and then load that file into Shiny. This can be done using a cron job.

**Comment by hmcclungiii** [comment]
I might recommend using the cron job to read the log and store the values in the database.

**Comment by Deleetdk** [comment]
Yes. There's an easy R function for reading the apache logs into a file. Then just set a cron job for that script once a day. The Shiny can then just load the latest version.
